src/__tests__/actions/productActions.test.ts

```diff
@@ -0,0 +1,106 @@
+import * as actions from '../../store/actions/productActions';
+import { IProduct } from '../../types/product';
+import { productActionTypes } from '../../store/actionTypes/productActionTypes';
+
+const tempProduct: IProduct = {
```

Comment:
This mock is using in two places, you can move to file or export from test reducer filer to this one.

Reply:
Done